### DISCUSSION TITLE
New Website - Open Web Calendar

### DISCUSSION THREAD

**Discussion by giscus** [niccokunzmann/open-web-calendar#328] [Announcements]
# New Website - Open Web Calendar

Embed a highly customizable web calendar into your website using ICal source links

https://open-web-calendar.quelltext.eu/blog/2024/04/10/new-website/

**Comment by niccokunzmann** [comment]
This is the first comment on the website post. I think, I should add that I am also quite grateful that I can use gisqus to comment on the project! And Thanks to NLNet and Gerben to propose that a project site is actually quite a useful thing.
